routes/helpers/render.js

```diff
@@ -61,6 +61,8 @@ let render = {
     vars.qualifiedURL = config.qualifiedURL;
 
     vars.urlPath = url.parse(req.originalUrl).pathname;
+    
+    vars.returnTo = req.get('Referer')
 
```

Comment:
This won't work. Say I give you a link from mycoolwebsite.example to create an account on lib.reviews. If we just directly check the referer, you'll now be sent back to mycoolwebsite.example. Generally, the returnto should under no circumstances send the user back to a page that's not on lib.reviews itself. This is why sites like Wikipedia pass a relative returnto on as a request parameter on each page that links to the login or register button.